src/nnet3/nnet-example-utils.cc

```diff
@@ -816,30 +823,43 @@ void UtteranceSplitter::GetGapSizes(int32 utterance_length,
 void UtteranceSplitter::GetChunksForUtterance(
     int32 utterance_length,
     std::vector<ChunkTimeInfo> *chunk_info) {
-  std::vector<int32> chunk_sizes;
-  GetChunkSizesForUtterance(utterance_length, &chunk_sizes);
-  std::vector<int32> gaps(chunk_sizes.size());
-  GetGapSizes(utterance_length, true, chunk_sizes, &gaps);
-  int32 num_chunks = chunk_sizes.size();
-  chunk_info->resize(num_chunks);
-  int32 t = 0;
-  for (int32 i = 0; i < num_chunks; i++) {
-    t += gaps[i];
-    ChunkTimeInfo &info = (*chunk_info)[i];
-    info.first_frame = t;
-    info.num_frames = chunk_sizes[i];
-    info.left_context = (i == 0 && config_.left_context_initial >= 0 ?
-                         config_.left_context_initial : config_.left_context);
-    info.right_context = (i == num_chunks - 1 && config_.right_context_final >= 0 ?
-                          config_.right_context_final : config_.right_context);
-    t += chunk_sizes[i];
+    int32 t = 0;
```

Comment:
I guess the main issue with how this looks is that you are using a 4-space indentation instead of 2-space. Please re-configure your editor and make sure the indentations are right.